src/test/scala/net/jcazevedo/moultingyaml/YamlPrettyPrinterSpec.scala

```diff
@@ -231,5 +231,53 @@ class YamlPrettyPrinterSpec extends Specification {
           |Ken Griff: null
           |""".stripMargin
     }
+
+    "pretty print according to a provided configuration" in {
```

Comment:
Can you update the name of this spec (it is testing the `print` method and not the `prettyPrint` method)?